src/wattsi.pas

```diff
@@ -305,7 +305,7 @@    TCrossReferences = record
                      HadSpace := True;
                   end;
                end;
-            '"', '?': ; // skipped silently
+            '"', '?', '`': ; // skipped silently
```

Comment:
Nice, I guess this means we no longer need hardcoded IDs for headings with HTTP headers in them. (Although putting explicit IDs in is probably still better...)